Tasks/AzureFileCopy/AzureFileCopyJob.ps1

```diff
@@ -52,7 +52,15 @@ param (
 
     Write-Verbose "Initiating copy on $fqdn " -Verbose
 
-    [String]$copyToAzureMachinesBlockString = "Copy-ToAzureMachines -MachineDnsName `$fqdn -StorageAccountName `$storageAccount -ContainerName `$containerName -SasToken `$sasToken -DestinationPath `$targetPath -Credential `$credential -AzCopyLocation `$azCopyLocation -AdditionalArguments `$additionalArguments -WinRMPort $winRMPort $cleanTargetPathOption $skipCACheckOption $httpProtocolOption $enableDetailedLoggingOption"
+    [String]$copyToAzureMachinesBlockString = [string]::Empty
+    if([string]::IsNullOrWhiteSpace($additionalArguments))
```

Comment:
Muthu, I agree on the refactoring. 
Our plan was to do the very minimal and simple change for the hot fix - that is easy to review too,